_posts/2022-02-16-reviewing-pull-requests.adoc

```diff
@@ -70,110 +69,106 @@ In the current state of the project, there are four endpoints:
 |Creates a Food resource
 |===
 
-It looks like a "colleague" has opened a new PR that adds a new `GET` endpoint, `/food/restaurant/{restaurantName}` that retrieves a list of all `Food` resources served from a specified restaurant:
+It looks like your colleague has opened a new PR that adds a new `GET` endpoint, `/food/restaurant/{restaurantName}`. The endpoint retrieves a list of all `Food` resources served from a specified restaurant:
 
 image::/assets/img/reviewing-pull-requests/pr.png[The pull request to review]
-Figure 1: The GitHub PR that we need to review.
+Figure 1: The GitHub PR you need to review.
 
-Let's review this PR by launching a new {prod} workspace. From the {prod} workspace, we can verify the PR by running unit tests, running the application, accessing the endpoint, building the application, all without leaving our web browser.
+Review this PR by launching a new {prod} workspace. Here you can verify the PR by running unit tests, running the application, accessing the endpoint, and building the application, all within your web browser.
 
-The workspace and IDE editor can be launched on link:https://www.eclipse.org/che/docs/che-7/hosted-che/hosted-che/[Eclipse Che Hosted by Red Hat] by clicking the {prod} badge from the base GitHub repository's `README`:
+Launch the workspace and Web IDE editor on link:https://www.eclipse.org/che/docs/che-7/hosted-che/hosted-che/[Eclipse Che Hosted by Red Hat] by clicking the {prod} badge from the base GitHub repository's `README`:
 
-image::/assets/img/reviewing-pull-requests/badge.png[Badge used to launch workspace]
-Figure 2: A badge from the base repository's `README.md` that launches a developer workspace when clicked.
+image::/assets/img/reviewing-pull-requests/badge.png[Badge used to launch workspaces]
+Figure 2: Clicking on a badge from the base repository's `README.md` launches a developer workspace.
 
 === Launching a workspace and reviewing the PR
 image::/assets/img/reviewing-pull-requests/ide.png[The Che-theia editor]
 Figure 3: The Che-Theia editor.
 
-Once the workspace has launched and the Web IDE has opened, you will see that the project has been cloned already (see the Explorer view on the bottom-right).
+After you launch the workspace, the web IDE will open automatically and you'll see that the project has been cloned already (see the Explorer view on the bottom-right).
```

Comment:
```suggestion
After you start the workspace, the web IDE will open automatically and you'll see that the project has been cloned already (see the Explorer view on the lower-right).
```

Reply:
This is a mistake on my part, instead of the `lower-right`, it should be on the left: https://github.com/dkwon17/blog/pull/1#discussion_r808050211